src/index.js

```diff
@@ -127,6 +127,14 @@ app.setAppUserModelId('com.marshallofsound.gpmdp.core');
 
     // and load the index.html of the app.
     mainWindow.loadURL(`file://${__dirname}/public_html/index.html`);
+
+    // minimize if 'start minimized' is on.
```

Comment:
I'm still around! Thanks for the help, @zevisert. I'll pull in the changes tomorrow and we can keep this PR rolling.